src/js/tracing/reactnativetracing.ts

```diff
@@ -85,6 +92,7 @@ const defaultReactNativeTracingOptions: ReactNativeTracingOptions = {
   ignoreEmptyBackNavigationTransactions: true,
   beforeNavigate: (context) => context,
   enableAppStartTracking: true,
+  enableNativeFramesTracking: true,
```

Comment:
Shouldn't this be false for now?

Reply:
We want to have everything enabled by default (when tracing is enabled) and only blanket disabled by a `enablePerformanceMonitoring: false` cc @bruno-garcia 

Will remove these individual options in a separate PR though.